README.md

```diff
@@ -0,0 +1,11 @@
+# Scratch Crossplane configuration
+
+This repository defines a "scratch" (or empty) [Crossplane configuration](https://docs.crossplane.io/v1.11/concepts/packages/#configuration-packages). This configuration is a starting point for users in [Upbound](https://console.upbound.io).
+
+## What's Inside
+
+A Crossplane configuration bundles a set of APIs. The [crossplane.yaml](https://github.com/upbound/configuration-scratch/blob/main/crossplane.yaml) is the definition file for a Configuration. This configuration has no APIs defined because it is from scratch.
```

Comment:
nitpick: consistency
```suggestion
A Crossplane configuration bundles a set of APIs. The [crossplane.yaml](https://github.com/upbound/configuration-scratch/blob/main/crossplane.yaml) is the definition file for a configuration. This configuration has no APIs defined because it is from scratch.
```